triple/node/node.go

```diff
@@ -74,25 +79,34 @@ func (n *Node) String() string {
 	return fmt.Sprintf("%s<%s>", n.t.String(), n.id.String())
 }
 
-// Parse returns a node given a pretty printed representation of Node.
+// Parse returns a node given a pretty printed representation of a Node or a BlankNode.
 func Parse(s string) (*Node, error) {
 	raw := strings.TrimSpace(s)
-	idx := strings.Index(raw, "<")
-	if idx < 0 {
-		return nil, fmt.Errorf("node.Parser: invalid format, could not find ID in %v", raw)
-	}
-	t, err := NewType(raw[:idx])
-	if err != nil {
-		return nil, fmt.Errorf("node.Parser: invalid type %q, %v", raw[:idx], err)
-	}
-	if raw[len(raw)-1] != '>' {
-		return nil, fmt.Errorf("node.Parser: pretty printing should finish with '>' in %q", raw)
-	}
-	id, err := NewID(raw[idx+1 : len(raw)-1])
-	if err != nil {
-		return nil, fmt.Errorf("node.Parser: invalid ID in %q, %v", raw, err)
+	switch raw[0] {
+	case slash:
+		idx := strings.Index(raw, "<")
+		if idx < 0 {
+			return nil, fmt.Errorf("node.Parser: invalid format, could not find ID in %v", raw)
+		}
+		t, err := NewType(raw[:idx])
+		if err != nil {
+			return nil, fmt.Errorf("node.Parser: invalid type %q, %v", raw[:idx], err)
+		}
+		if raw[len(raw)-1] != '>' {
+			return nil, fmt.Errorf("node.Parser: pretty printing should finish with '>' in %q", raw)
+		}
+		id, err := NewID(raw[idx+1 : len(raw)-1])
+		if err != nil {
+			return nil, fmt.Errorf("node.Parser: invalid ID in %q, %v", raw, err)
+		}
+		return NewNode(t, id), nil
+	case underscore:
+		id := ID(raw[2:len(raw)])
+		t :=  Type("/_")
+		return NewNode(&t, &id), nil
```

Comment:
return NewNode(&t, id), nil

Both changes for consistency with the code below. I would also suggest use NewType.